web/html/src/components/panels/InnerPanel.tsx

```diff
@@ -5,26 +5,41 @@ import { SectionToolbar } from "components/section-toolbar/section-toolbar";
 type Props = {
   title: string;
   icon: string;
-  buttons: React.ReactNode;
-  buttonsLeft?: React.ReactNode;
+  buttons: React.ReactNode[];
+  buttonsLeft?: React.ReactNode[];
   children: React.ReactNode;
 };
 
 function InnerPanel(props: Props) {
+  let toolbar;
+
+  if (props.buttons?.length > 0 || (props.buttonsLeft !== undefined && props.buttonsLeft?.length > 0)) {
+    toolbar =
+      <SectionToolbar>
+        {
+          props.buttonsLeft !== undefined && props.buttonsLeft?.length > 0 ?
```

Comment:
The undefined check should be superfluous with the optional chaining in the next part of the condition.

Reply:
It way Typescript complaining about it: `Object is possibly undefined` :thinking:

Reply:
Ah okay I see, my bad sorry. When `buttonsLeft` is `undefined`, `props.buttonsLeft?.length` will be undefined and Typescript is unhappy about the `undefined > 0` comparison. The problem will go away if you check for `!== 0` instead.

Reply:
Thanks for the hint, I fixed it accordingly :wink: